lnwallet/test/test_interface.go

```diff
@@ -1685,6 +1685,31 @@ func newTx(t *testing.T, r *rpctest.Harness, pubKey *btcec.PublicKey,
 	return tx1
 }
 
+// testGetTransactionDetails checks that GetTransactionDetails returns the
+// correct amount after a transaction has been sent from alice to bob.
+func testGetTransactionDetails(r *rpctest.Harness,
+	alice, bob *lnwallet.LightningWallet, t *testing.T) {
+
+	const txFee = int64(14500)
+
+	bobPkScript := newPkScript(t, bob, lnwallet.WitnessPubKey)
+	txFeeRate := chainfee.SatPerKWeight(2500)
+	amountSats := btcutil.Amount(btcutil.SatoshiPerBitcoin - txFee)
+	output := &wire.TxOut{
+		Value:    int64(amountSats),
+		PkScript: bobPkScript,
+	}
+	tx := sendCoins(t, r, alice, bob, output, txFeeRate, true, 1)
+	txHash := tx.TxHash()
+	assertTxInWallet(t, alice, txHash, true)
+	assertTxInWallet(t, bob, txHash, true)
+
+	txDetails, err := bob.GetTransactionDetails(&txHash)
+	require.NoError(t, err, "unable to receive transaction details")
+	require.Equal(t, txDetails.Value, amountSats, "expected transaction "+
```

Comment:
There are formatting placeholders in the last argument, but `Equalf()` wasn't used. IMO this can just be `require.Equal(t, txDetails.Value, amountSats, "tx value")` since the required library will produce a nice error message already.

Reply:
@guggero thanks for the review. Will finish it soon.